sway/input/seat.c

```diff
@@ -291,8 +291,8 @@ void sway_seat_configure_xcursor(struct sway_seat *seat) {
 		seat->cursor->cursor->y);
 }
 
-void sway_seat_set_focus(struct sway_seat *seat,
-		struct sway_container *container) {
+void _sway_seat_set_focus(struct sway_seat *seat,
```

Comment:
This should be `static`

Reply:
Err, this is confusing. Maybe pick another name?

Reply:
nah